fuse/pathfs/pathfs.go

```diff
@@ -276,7 +276,6 @@ func (n *pathInode) GetPath() string {
 	// them, them, but since this is a hot path, we take some
 	// effort to avoid allocations.
 
-	n.pathFs.pathLock.RLock()
```

Comment:
oh, ah! 
I misunderstood your commit message, looked at the code and then removed this lock too.

Can you rebase?